zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/repo/zeppelinhub/security/Authentication.java

```diff
@@ -193,7 +196,16 @@ private String decrypt(String value, String initVector) {
   }
 
   private Key generateKey() {
-    return new SecretKeySpec(toBytes(KEY), CIPHER_ALGORITHM);
+    try {
+      KeyGenerator kgen = KeyGenerator.getInstance(CIPHER_ALGORITHM);
+      kgen.init(128, new SecureRandom(toBytes(KEY)));
+      SecretKey secretKey = kgen.generateKey();
+      byte[] enCodeFormat = secretKey.getEncoded();
+      return new SecretKeySpec(enCodeFormat, CIPHER_ALGORITHM);
+    } catch (Exception e) {
+      e.printStackTrace();
```

Comment:
Then, it would be nice to use `Logger` instead of `e.printStackTrace();` except for it, LGTM.